tests/Hyperwallet/Tests/HyperwalletTest.php

```diff
@@ -13,24 +13,28 @@
 use Hyperwallet\Model\BankCardStatusTransition;
 use Hyperwallet\Model\PaperCheck;
 use Hyperwallet\Model\PaperCheckStatusTransition;
-use Hyperwallet\Model\Payment;
-use Hyperwallet\Model\PaymentStatusTransition;
+use Hyperwallet\Model\Transfer;
+use Hyperwallet\Model\TransferRefund;
+use Hyperwallet\Model\TransferStatusTransition;
 use Hyperwallet\Model\PayPalAccount;
 use Hyperwallet\Model\PayPalAccountStatusTransition;
+use Hyperwallet\Model\Payment;
+use Hyperwallet\Model\PaymentStatusTransition;
 use Hyperwallet\Model\PrepaidCard;
 use Hyperwallet\Model\PrepaidCardStatusTransition;
-use Hyperwallet\Model\Transfer;
 use Hyperwallet\Model\TransferMethod;
-use Hyperwallet\Model\TransferStatusTransition;
 use Hyperwallet\Model\User;
 use Hyperwallet\Model\VenmoAccount;
 use Hyperwallet\Model\VenmoAccountStatusTransition;
 use Hyperwallet\Response\ErrorResponse;
 use Hyperwallet\Util\ApiClient;
 
-class HyperwalletTest extends \PHPUnit_Framework_TestCase {
+class HyperwalletTest extends \PHPUnit_Framework_TestCase
+{
+    private $includeIntegrationTest = false; //change this value to true if integration tests have to be run
 
-    public function testConstructor_throwErrorIfUsernameIsEmpty() {
+    public function testConstructor_throwErrorIfUsernameIsEmpty()
```

Comment:
Revert the changes here. Pls revert all the formatting changes done in this whole file.

Reply:
Done